.github/workflows/vulncheck.yml

```diff
@@ -0,0 +1,20 @@
+on: [push, pull_request_target]
+name: Vulnerability Check
+jobs:
+  Security:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: 1.19.x
```

Comment:
Shouldn't this run with 1.16.x and 1.18.x too?

Reply:
Benchmark also runs on 1 go version. I think we don't need another versions in this workflow